checklist/checklist_sys.csv/tests.csv

```diff
@@ -0,0 +1,9 @@
+ID,Topic,Title,Requirement,Explanation,References
+2.1,Data Presence,Test Data Fetching and File Reading,"Verify that the data fetching API or data file reading functionality works correctly. Ensure that proper error handling is in place for scenarios such as missing files, incorrect file formats, and network errors.","Ensure that the code responsible for fetching or reading data can handle errors. This means if the file is missing, the format is wrong, or there's a network issue, the system should not crash but should provide a clear error message indicating the problem.",(general knowledge)

```

Comment:
Under the topic of "data presence" (or saving and loading data) - At the beginning of a data analysis pipeline I would expect two possible functions: 

1. A data retrieval function (e.g., downloads data files from a url, reads data from a database, requests data from an API, screen scrapes data from a website).

2. A saving/writing the retrieved data to disc function, which takes the data retrieved into memory and writes it to disc.

These might be general knowledge, but these were also written in the Microsoft guide, and so I think that should be listed as a reference. *Note - not all projects may have these functions. For example some projects data might have been collected manually.*

I think for all test suites, for any function, I would expect the following three buckets of test cases:
- Simple expected use cases
- Edge cases (unexpected, or rare use cases)
- Errors

Reply:
> I think for all test suites, for any function, I would expect the following three buckets of test cases:
> Simple expected use cases
> Edge cases (unexpected, or rare use cases)
> Errors
Agree